documentation/proc-pages/installation/installation-ubuntu.md

```diff
@@ -43,7 +43,7 @@ git clone https://github.com/ukaea/PROCESS
 cd PROCESS
 ```
 
-Create and activate a virtual environment:
+If you would like to install PROCESS using a package manager such as conda, follow the instructions for [HPC installation](https://ukaea.github.io/PROCESS/installation/csd3-for-process/). Otherwise, create and activate a virtual environment. Note that you must use Python 3.8 - 3.11:
```

Comment:
```suggestion
If you would like to install PROCESS using a package manager such as conda, follow the instructions for [HPC installation](https://ukaea.github.io/PROCESS/installation/csd3-for-process/). Otherwise, create and activate a virtual environment, as shown below. Note that you must use Python 3.10 or 3.11:
```